Assets/Scripts/MQ/Queue.cs

```diff
@@ -22,25 +24,27 @@ public class RemoteQueue : Queue
         public string targetQueueName;
         public string targetQmgrName;
         public string transmissionQueueName;
+        public new const int currentDepth = 0;
```

Comment:
any reason for `new` to be here?

Reply:
`new` here tells the compiler to hide the base field of currentDepth and use the child class's value instead.

Reply:
nice!